metashare/api/models.py

```diff
@@ -315,6 +362,20 @@ def get_serialized_representation(self):
 
     # end PushMixin configuration
 
+    # begin CreatePrMixin configuration:
+    create_pr_event = "TASK_PR_CREATE"
+
+    def get_repo_id(self, user):
+        return self.project.repository.get_repo_id(user)
+
+    def get_base(self):
+        return self.branch_name
+
+    def get_head(self):
+        return self.project.branch_name
```

Comment:
These are backwards, I think. `base` should be the project branch, `head` should be the task branch.